library/Fission/User/Modifier/Class.hs

```diff
@@ -4,9 +4,12 @@ module Fission.User.Modifier.Class
   ) where
 
 import           Database.Persist as Persist
-import           Network.IPFS.CID.Types
 import           Servant.Server
 
+import qualified Network.IPFS.Add.Error as IPFS.Pin
+import qualified Network.IPFS.Get.Error as IPFS.Stat
```

Comment:
Stat or Get? Looks like both modules exist 🤔

Reply:
There's not `Stat.Error`

`Stat` returns the same type of errors as `Get` (in the same way that `Pin` returns the same type of errors as `Add` if you look at the prev input)

These could probably use a renaming in the ipfs pkg :thinking: maybe `Modify.Error` & `Retrieval.Error` since pins & adds are both "modifies" & gets & stats are both "retrievals". and neither of those words can be confused with actual ipfs commands

Reply:
Okay, cool cool. Just wanted to make sure it was right. Maybe add a note. No need to change on the IPFS side right now 👍